Solving guidance page (#23777)
sympy/sympy: doc/src/guides/solving/solve-equation-algebraically.md

### Include the variable to be solved for in the function call

We recommend you include the variable to be solved for as the second argument

> for either function. While this is optional for equations with a single symbol,

Reviewer: ```suggestion
for either function. While this is optional for univariate equations,
```

it is a good practice because it ensures SymPy will solve for the desired

> symbol. For example, you may expect the following to solve for $x$, and SymPy
> will solve for $y$:

Reviewer: ```suggestion
symbol. For example, you might be interested in a solution for $x$, but SymPy
solves for $y$:
```

### Ensure consistent formatting from {func}`~.solve` by using `dict=True`

> {func}`~.solve` produces various output formats depending on the answer, unless
> you use `dict=True` to ensure the result will be formatted as a dictionary. We
> recommend using `dict=True`, especially if you want to extract information from
> the result programmatically.

Reviewer: ```suggestion
{func}`~.solve` produces a variety of output as explained in :ref:`solve_output`. Using
`dict=True` will give a consistent output format which is especially important when
extracting information about the solution programmatically.
```

### Put your equation into `Eq` form

Put your equation into `Eq` form, then solve the `Eq`. This approach is
convenient if you are interactively solving an equation which you already have

> in the form of an equation, or which you think of as an equality.

Reviewer: ```suggestion
in the form of an equation, or which you think of as an equality. It also
help to prevent sign errors when subtracting one side from the other.
```

2*pi
```

{func}`~.solve` tries to return just enough solutions so that all (infinitely
many) solutions can generated from the returned solutions by adding integer

> multiples of the {func}`~.periodicity` of the equation, here $2\pi$.

Reviewer: nice reference there to `periodicity`
